features/fields.feature

```diff
@@ -0,0 +1,341 @@
+      | geslachtsaanduiding.code         | M         |
+      | geslachtsaanduiding.omschrijving | man       |
+
+    Scenario: Gevraagde velden zijn gegevens van één gegevengroep van de persoon
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                |
+      | type                | RaadpleegMetBurgerservicenummer                       |
+      | burgerservicenummer | 999995078                                             |
+      | fields              | burgerservicenummer,naam.geslachtsnaam,naam.voornamen |
+      Dan heeft de persoon met burgerservicenummer '999995078' alleen de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999995078 |
+      En heeft de persoon met burgerservicenummer '999995078' alleen de volgende 'naam' gegevens
+      | naam          | waarde   |
+      | geslachtsnaam | Groenen  |
+      | voornamen     | Franklin |
+
+    Scenario: Gevraagde velden zijn gegevens van meerdere gegevensgroepen van de persoon
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                          |
+      | type                | RaadpleegMetBurgerservicenummer                                 |
+      | burgerservicenummer | 999995078                                                       |
+      | fields              | burgerservicenummer,geslachtsaanduiding.code,naam.geslachtsnaam |
+      Dan heeft de persoon met burgerservicenummer '999995078' alleen de volgende gegevens
+      | naam                     | waarde    |
+      | burgerservicenummer      | 999995078 |
+      | geslachtsaanduiding.code | M         |
+      En heeft de persoon met burgerservicenummer '999995078' alleen de volgende 'naam' gegevens
+      | naam          | waarde  |
+      | geslachtsnaam | Groenen |
+
+    Scenario: Gevraagd veld is een gegeven van een gegevensgroep collectie van de persoon
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                         |
+      | type                | RaadpleegMetBurgerservicenummer                                |
+      | burgerservicenummer | 999995078                                                      |
+      | fields              | burgerservicenummer,partners.type,partners.burgerservicenummer |
+      Dan heeft de persoon met burgerservicenummer '999995078' alleen de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999995078 |
+      En heeft de persoon met burgerservicenummer '999995078' een 'partner' met alleen de volgende gegevens
+      | naam                | waarde    |
+      | type                | Partner   |
+      | burgerservicenummer | 999992935 |
+
+    Abstract Scenario: Gevraagd veld heeft geen waarde voor de persoon

```

Comment:
Moeten betreffende velden dan niet ook met een lege waarde in de tabel die bovenaan dit bestand bij 'Achtergrond' staat wordt opgenomen?

Reply:
denk ik ook. toegevoegd

Reply:
ik heb nog een voorbeeld toegevoegd en een opmerking die het voorbeeld uitlegt
```
      | geheimhoudingPersoonsgegevens   | geheimhoudingPersoonsgegevens wordt niet opgenomen bij waarde 0 |
      | partners.geboorte.plaats        | plaats wordt niet opgenomen bij standaardwaarde 0000            |
```